app/webroot/js/sentences.show_another.js

```diff
@@ -30,6 +30,13 @@ $(document).ready(function(){
 });
 
 function loadRandom(lang){
-    $(".random_sentences_set").html("<div class='block-loader loader'></div>");
-    $(".random_sentences_set").load("/sentences/random/" + lang);
+
+    //$(".random_sentences_set").html("<div class='block-loader loader'></div>");
+    $.get( "/sentences/random/" + lang, function( data ) {  
+        if($(data).find(".mainSentence > .content > .sentenceContent").text() == "") {
+            alert("There was an error in performing this function. Please try later.")
```

Comment:
There is no need to use `alert()` here. Besides, raw strings in javascript files can’t be localized so it’s a no-go. I think you can just return the error as HTML, just like the random sentence is returned.